services/provider/server/consumer.go

```diff
@@ -83,6 +84,10 @@ func (c *ocsConsumerManager) Create(ctx context.Context, onboard ifaces.StorageC
 		},
 	}
 
+	if onboardingTicket != nil {
+		consumerObj.Spec.StorageQuotaInGiB = int(onboardingTicket.StorageQuotaInGiB)
+	}
+
```

Comment:
Instead of an if outside of the struct you can do:
```
Spec: ocsv1alpha1.StorageConsumerSpec{
			Enable: false,
			StorageQuotaInGiB: ptr.Deref(onboardingTicket.StorageQuotaInGiB, 0),
		},
```